src/components/dar_table/DarElectionRecords.js

```diff
@@ -34,7 +34,7 @@ const goToReviewResults = (dar, history, status) => {
 };
 const electionStatusTemplate = (consoleType, dar, election, recordTextStyle, votes, showVotes, history) =>{
   const linkedStatuses = ['Unreviewed', 'Approved', 'Denied', 'Canceled'];
-  const status = election ? processElectionStatus(election, votes, showVotes) : 'Unreviewed';
+  const status = processElectionStatus(election, votes, showVotes);
```

Comment:
not a functional change, but since I added a null check to processElectionStatus this is no longer necessary